scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -87,6 +112,15 @@ Param (
     [ValidateNotNullOrEmpty()]
     [string] $EventHubConnectionString = $(Throw "Event hub connection string is required"),
 
+    [ValidateNotNullOrEmpty()]
+    [string] $EdgeE2ERootCACertRSAFile = $NULL,
```

Comment:
Do you need to explicitly set to $NULL? I think $NULL is default value for string.

Reply:
Sure I am a bit more on the declarative side so I added this. Seems to be a pattern used else where in this file as well.